src/components/SetupTab/index.jsx

```diff
@@ -1,89 +1,83 @@
 import React from 'react';
-import { remote } from 'electron';
 import PropTypes from 'prop-types';
 
 import Container from 'react-bootstrap/Container';
 import Row from 'react-bootstrap/Row';
 import Col from 'react-bootstrap/Col';
 import Button from 'react-bootstrap/Button';
-import Form from 'react-bootstrap/Form';
 import DropdownButton from 'react-bootstrap/DropdownButton';
 
-import { ArgInput } from './ArgInput';
-import { SaveParametersButton, SavePythonButton } from '../SaveDropdown'
-import { fetchDatastackFromFile, fetchValidation, 
-         saveToPython } from '../../server_requests';
-import { argsDictFromObject, boolStringToBoolean } from '../../utils';
-
-
+import ArgsForm from './ArgsForm';
+import SaveFileButton from '../SaveFileButton';
+import { fetchValidation, saveToPython } from '../../server_requests';
+import { argsDictFromObject } from '../../utils';
+
+/** Toggle properties that control the display of argument inputs.
+ *
+ * This function returns a copy of the SetupTab argsValues object
+ * after updating the 'ui_option' property of any arguments listed
+ * in the 'ui_control' array of `argsSpec[argkey]`.

```

Comment:
Does `jsdoc` assume any particular markup language within docstrings?  Markdown?

Reply:
> Does `jsdoc` assume any particular markup language within docstrings? Markdown?

It looks like there's a plugin for Markdown rendered to html. But I haven't been concerned with any of that thus far. So far it doesn't feel like we're developing an API that would warrant rendered docs.